addon/components/au-modal.js

```diff
@@ -2,6 +2,8 @@ import { assert } from '@ember/debug';
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 
+const FOCUS_TRAP_ADDITIONAL_ELEMENTS = ['#ember-basic-dropdown-wormhole'];
```

Comment:
Should this perhaps be an application variable/option?

Reply:
What did you have in mind exactly? I'm not sure what you mean.

Reply:
Nevermind that last idea.

Reply:
Ideally apps could simply wrap the AuModal component and configure any app wide configs that way, but that isn't really possible yet with named blocks..

So a different way to configure this might be nice. The problem is that the configs we currently support are only temporary (until v3) and the same system doesn't work for v2 addons so I would rather not add extra options that way.

Looking up the environment config could work, but that's not really a good practice (anymore). I would have to think about how this can best be done but I think we can merge this as-is and add the option in a future PR.